libs/designsystem/modal/src/modal/footer/modal-footer.component.scss

```diff
@@ -43,6 +43,7 @@ ion-footer {
 :host(.inline) ion-footer {
   background: transparent;
   box-shadow: none;
+  @include themes.apply-light-theme;
```

Comment:
This only fixes it for the inline modal but would be good to fix it for the `light` themeColor footer, by also applying the mixin to the `:host(.light)` selector 👍